### ISSUE TITLE
What if I have multiple Daily Notes from the same day?

### ISSUE THREAD

**Issue by Unvented9039** [Kageetai/obsidian-plugin-journal-review#12]
I've been importing journal entries from the Journey app. As a result, I have Daily Notes with the date _and_ the time in the title. Your plugin is only displaying the first note from the day in history. Is there any way to get all of the day's notes to display? 

This is so close to replicating Journey's Throwback feature.

**Comment by Kageetai** [comment]
Hey, glad to hear you are using my plugin for that. I actually also imported my notes from Journey and stumbled upon a similar problem.
I only support the same note structure also supported by the standard daily/periodic notes, which to my knowledge does not support multiple notes per day. E.g the "open daily note" command can only open a single note per day, no?
I can try to parse several notes per day, but that's rather complex and not sure when I'd get to that, so no quick fix for that I think.

Out of curiosity, how did you import your notes from Journey to Obsidian? I had to write a short custom script, but it didn't work perfectly.

**Comment by Unvented9039** [comment]
Hi Michael, and thanks for the reply.

That's funny that you're trying to import Journey notes. Is that what
prompted you to create this plugin?

I used a Journey to Obsidian converter which I found out about here:
https://disordered.me/notes/my-migration-from-journey-app-to-obsidian/

In Obsidian, I changed the daily note date format to include hours, minutes
and seconds. Then, when I ran the converter, I had to specify that I wanted
the output to show hours, minutes and seconds as well. It worked well
enough.

Ultimately, I just decided to keep using Journey though.

**Comment by Kageetai** [comment]
Yes, I had been using Journey for a few years, but after starting to use Obsidian and liking it more and more, I thought I slowly wanna switch over to a platform where I have more control and thought it a good opportunity to get my hands dirty on some plugins for this wonderful software.

I see your use case now, but would need to investigate how to achieve this.

What made you stick with Journey in the end? What was missing for you in Obsidian?

**Comment by Unvented9039** [comment]
Yeah, having more control was a big plus for me too. And I'd been paying
for a subscription to Journey, and I didn't want to do that anymore. There
are some things about Journey that I don't think work right, so I was
looking for a perfect experience.

In the end, I simply thought Journey did journaling better. One thing that
was missing from Obsidian is automation. I have Journey pulling in a few
RSS feeds that show what media I'm consuming. I use Trakt, for example, to
log my TV and movie watching. When I do that there, Journey automatically
creates an entry. I love to look back and see what I was watching and
reading. Obsidian has two RSS plugins that I could find, but neither of
them work the way I want them to.

Obsidian lacks the slick Throwback feature that Journey has, which is
another thing I love. So, I had high hopes when I found your plugin! It's a
great plugin, but again, just doesn't work the way I want it to.

I don't like how Obsidian's desktop and Android apps feel like two separate
experiences. I was able to get syncing going for free, but I disliked
having my entire vault on my phone's storage.

Those are the big 3. There are a few little things about Journey that I was
willing to sacrifice, such as the ability to create a post using a photo
and the date and other metadata are automatically changed to reflect when
the photo was taken.

Journey is rolling out an AI feature, too, which looks pretty useful.

I'd never used Obsidian before I went down this road, so I had a lot to
learn and figure out. I think if I were already using Obsidian for other
things, then it would make sense to use it for journaling.

I also feel like Obsidian is for people who really like to tinker and know
how to write and edit code, use command line interface, and so on. And none
of that describes me! 😆 I had to get Chat GPT to help me run that Journey
to Obsidian converter.

Having said all that, I do like Obsidian and maybe I'll have a use for it
someday.

Whew! Sorry for the long boring reply!

Have you completely moved away from Journey then? Using Obsidian for
journaling?

**Comment by Kageetai** [comment]
Ah, I have never even tried the subscription, but makes sense that you are using even more features specific to Journey. I also found some of their features just a bit more restrictive, but otherwise a great product nonetheless.

Regarding the automations, as I have never tried them with Journey, I cannot speak for them, but actually thought it seems easier to have more powerful and flexible automations with Obsidian, as I can have the choice in plugins and such. But of course, it's also more of a chore potentially, to set up everything manually. As a developer myself, I don't usually mind it that much.

With that said, the biggest thing I was also missing was the throwback feature, so I thought to try making it myself.
If there is anything, apart from initial problem with multiple entries per day, that you mentioned, that you would miss here from Journey, I can add it to my to-do list ;)
Regarding adding photos and automatically use the metadata, I had also thought of that and considered using it, but for now I am just using the [Obsidian Geocode Properties](https://github.com/jose-elias-alvarez/obsidian-geocoding-properties) plugin for some of that.

Oh, and might I ask how you got the syncing to work for free? I am paying for Obsidian sync currently and did try some free methods, but nothing seemed to work perfectly yet, so decided to pay for the official solution for the time being.

Therefore, as you said, Obsidian is definitely more of a "tinker tool", which I enjoy, but I understand the hurdles that creates. I think the team tries to strive for a good balance for a majority of users, though. I enjoy that power of really "making the tool my own" :)

I haven't entirely moved away from Journey yet, mainly to see whether my plugin catches all past entries correctly and the same ones Journey would show me for a single day. So for the time being, I use both in parallel.

And never be sorry for a nice message like this, I am enjoying discussing a software I like and its features or something I am working on etc :)

**Comment by Unvented9039** [comment]
I thought I'd have a better experience with automation plugins for
Obsidian. Like you said, I figured they'd be more powerful and flexible
than what I have with Journey. But, if that experience exists, I was unable
to find it. Unlike you, I'm not a developer in any way shape or form, so
I'm pretty much at the mercy of the plugin developer. I imagine it's very
satisfying to tinker until you get it just the way you want it. So, I can
understand what you're feeling.

Regarding syncing, I use an Android device and found an app called
DriveSync. I have my Obsidian desktop vault in a folder on Google Drive and
that app simply watches for changes to that folder. It works very well.
But, as I mentioned, the downside was that I had my entire vault on my
phone since Obsidian only uses local Android storage. (As far as I know.)

**Comment by Kageetai** [comment]
I understand, finding and settings up all the specific features for a personal use case, can be cumbersome. I had the idea to create a "bundle of plugins" to provide a more fully-fleshed journaling experience and approximate something like Journey at some point. But that's a rather far future idea :D

Interesting, I have also tried apps like DriveSync (or FolderSync for me), but the problem triggering a new sync on the phone and the files on my phone would often be outdated, as no tool I found would watch the files on the phone and sync them automatically to Google Drive when I changed something reliably.
Is this the [DriveSync](https://play.google.com/store/apps/details?id=com.ttxapps.drivesync&hl=en&gl=US) you mean?

**Comment by Unvented9039** [comment]
That bundle idea sounds great! Seems like a daunting task to be sure.

And yes, that app you linked to is the one. Funny how the app is titled
differently in the Play Store listing. The app when installed is titled
DriveSync.
